feat: simplified launch of environments

lorenzo-cavazzi opened the pull request:
This PR adds a quick-start button on the project header to quickly start a new session from the latest commit on the default branch and connect to it directly.
It doesn't consider autosaves or currently running sessions _yet_, but that will come soon with #1345 , significantly improving the experience when users push new commits and then try to start another session.

P.S. The button in the header looks misaligned on smaller screens, but the header is going to change significantly with #1385

![Screenshot_20210624_184601](https://user-images.githubusercontent.com/43481553/123302880-7514d700-d51d-11eb-944d-098a9e1186ce.png)

![Screenshot_20210624_184625](https://user-images.githubusercontent.com/43481553/123302854-6d553280-d51d-11eb-9e3d-162f68e78fa5.png)

/deploy renku=ui-design-2021-startbutton
fix #1152

ciyer reviewed (changes requested):
We have been waiting for this feature for a while; the experience of using Renku is much nicer now! 

The code is good, I have only a small request to change wording, but I am not 100% what the better wording is.

And a small cosmetic thing -- it looks like there are two spinners on this page, which is a bit weird. Can we have just one spinner?

![image](https://user-images.githubusercontent.com/1196411/123640334-5a927480-d821-11eb-9901-2b7e88b12f8f.png)

lorenzo-cavazzi commented on client/src/notebooks/Notebooks.present.js:
```diff
@@ -1056,25 +1067,82 @@ class StartNotebookServer extends Component {
+  let progress = 0;
+  let message = "";
+  if (fetching.notebooks) {
+    message = "Starting Jupyter";
```
That step is about checking the running pods.
I tried to avoid technical words and I ended up with a somewhat wrong message. What about "Checking running pods?" Or just "Checking pods"?

rokroskar replied:
"checking running sessions"?

lorenzo-cavazzi replied:
That sounds reasonable

lorenzo-cavazzi commented:
I changed the message and removed the double loader. if you think `Checking pods` is not a good definition, feel free to suggest something else (or directly change the text in the code on this PR)

![Screenshot_20210628_161128](https://user-images.githubusercontent.com/43481553/123652220-646da500-d82c-11eb-9876-6e604e3504b4.png)

![Screenshot_20210628_161142](https://user-images.githubusercontent.com/43481553/123652232-66cfff00-d82c-11eb-99ba-114702258789.png)

ciyer reviewed (approved):
Woohoo! 🎉

ciyer commented:
This was included in #1385.
